python/lsst/meas/algorithms/sourceSelector.py

```diff
@@ -97,3 +103,301 @@ def _isBad(self, source):
 sourceSelectorRegistry = pexConfig.makeRegistry(
     doc="A registry of source selectors (subclasses of BaseSourceSelectorTask)",
 )
+
+
+class ColorLimit(pexConfig.Config):
+    """Select sources using a color limit
+
+    This object can be used as a `lsst.pex.config.Config` for configuring
+    the limit, and then the `apply` method can be used to test sources in
+    a catalog against the configured limit.
+
+    We refer to 'primary' and 'secondary' flux measurements; these are the
+    two components of the color, which is 'mag(primary) - mag(secondary)'.
```

Comment:
I think the doc should say something like `instFluxToMag(cat[primary]) - instFluxToMag(cat[secondary])`, to make it explicit what the conversion is.